01-main/packages/opensnitch

```diff
@@ -0,0 +1,9 @@
+DEFVER=1
+get_github_releases "evilsocket/opensnitch" "latest"
+if [ "${ACTION}" != "prettylist" ]; then
+    URL="$(grep "browser_download_url.*\.deb\"" "${CACHE_FILE}" | grep -v 'python3-opensnitch-ui' | head -n1 | cut -d'"' -f4)"
```

Comment:
fails because the api lists the assets of the release and the latest release is only the UI - there's a link in the release page notes to the latest opensnitch deb but presumably if there's a release of opensnitch without the ui at any point there will be the reverse problem.  I think the only possibility would be to not use get_github_releases but to get the latest release as a web page and parse the link there for each component.  Not ideal but ...